code/code/misc/limits.cc

```diff
@@ -467,6 +467,22 @@ int TPerson::hitGain()
   if (affectedBySpell(SPELL_ENLIVEN))
     gain *= 2;
 
+  // Scale regen based on the number of bloodlust stacks 
+  if (affectedBySpell(SKILL_BLOODLUST)) {
+    int mod = 1;
```

Comment:
Why not zero?

Reply:
Probably should be in hindsight.  Updated.